src/tests/unit/tests/DetailsView/components/issue-filing-dialog.test.tsx

```diff
@@ -66,23 +66,18 @@ describe('IssueFilingDialog', () => {
             selectedBugFilingServiceData: selectedServiceData,
             bugFileTelemetryCallback: telemetryCallbackMock.object,
         };
-
-        isSettingsValidMock.setup(isSettingsValid => isSettingsValid(selectedServiceData)).verifiable();
-        createBugFilingUrlMock
-            .setup(createBugFilingUrl => createBugFilingUrl(selectedServiceData, selectedBugDataStub, envInfo))
-            .verifiable();
     });
 
-    it('render open', () => {
-        const testSubject = shallow(<IssueFilingDialog {...props} />);
+    it.each([true, false])('isSettingsValid: %s', isSettingsValid => {
+        isSettingsValidMock
+            .setup(isValid => isValid(selectedServiceData))
+            .returns(() => isSettingsValid)
+            .verifiable();
 
-        isSettingsValidMock.verifyAll();
-        createBugFilingUrlMock.verifyAll();
-        expect(testSubject.getElement()).toMatchSnapshot();
-    });
+        createBugFilingUrlMock
+            .setup(createBugFilingUrl => createBugFilingUrl(selectedServiceData, selectedBugDataStub, envInfo))
```

Comment:
should this return a value? I think it needs to and then you probably don't need the verifiable as the result is being use as the href for `ActionAndCancelButtonsComponent`. If this props shows up on the snapshot you can drop the verify; if it doesn't shows up then you probably want to keep it.

Reply:
So.... does the returned value make a difference on the snapshot?

Reply:
Sorry, yes the url appears in the snapshot.